Makefile

```diff
@@ -52,6 +52,8 @@ ifneq ($(KIND_VERSION), latest)
 else
 	KIND_ARGS =
 endif
+# KubeBuilder configuration
+KBVERSION := 2.3.1
```

Comment:
I'm wondering about bumping the version here (since it was originally set to `2.3.0`), but a minor version bump is probably okay. 🙂

Reply:
@dhaiducek good catch, I'll just set it to 2.3.0 - better to be safe for a backport